elements/rh-card/docs/10-style.md

```diff
@@ -174,3 +182,90 @@ become thinner, the container padding decreases.
 <uxdot-example width-adjustment="360px">
   <img src="../card-spacing-mobile.svg" alt="Card spacing on mobile">
 </uxdot-example>
+
+## Variants
+
+### Promo
+
+Promos are available in four different configurations.
+
+#### Standard promo
+
+A Standard promo also has a subtle border, but it uses the `Lighter` or `Darker` surface color. It does not have an image slot and the call to action appears either to the left or right of the text.
+
+<uxdot-example>
+  <img src="../card-variants-promo-standard.svg" alt="Standard promo with placeholder text">
+</uxdot-example>
+
+#### Featured promo
+
+A Featured promo uses the `Lightest` or `Darkest` surface color with a subtle border. An image can appear on the left or right side of a promo, but a featured promo can also omit an image.
+
+<uxdot-example>
+  <img src="../card-variants-promo-featured.svg" alt="Featured promo with placeholder text and image">
+</uxdot-example>
+
+#### Full-width promo
+
+A Full-width promo bleeds to the left and right edges of the screen. It does not have a border. Similarly to a featured promo, a full-width promo can use either the `Lightest` or `Darkest` surface color, and it can have no image or an image positioned to the left or right of the text.
+
+<uxdot-example variant="full">
+  <img src="../card-variants-promo-full-width.svg" alt="Full-width promo with placeholder text and image">
+</uxdot-example>
```

Comment:
same as above add width and height to img and width-adjustment to uxdot-example to match